app/src/main/java/nerd/tuxmobil/fahrplan/congress/contract/BundleKeys.java

```diff
@@ -12,6 +12,12 @@
     String ALARM_START_TIME =
             "nerd.tuxmobil.fahrplan.congress.ALARM_START_TIME";
 
+    // Lecture alarm notification
```

Comment:
Side note: There's no separation of concerns here. It doesn't feel like there's a good reason to keep all of these in one place.

Reply:
Agreed. Also naming is not very consistent.